setup.py

```diff
@@ -5,13 +5,15 @@
 install_requires = [
     'torch>=1.3.0',
     'gym>=0.9.7',
-    'numpy>=1.10.4',
+    'numpy>=1.11.0',
     'pillow',
     'filelock',
 ]
 
 test_requires = [
     'pytest',
+    'scipy',
```

Comment:
The tests use it.
https://github.com/pfnet/pfrl/blob/b29533b77c82f88fbc074714274ee8aa63dc270f/tests/utils_tests/test_random.py#L6